### ISSUE TITLE
Does not catch $VARIABLE = value

### ISSUE THREAD

**Issue by tripleee** [koalaman/shellcheck#986]
#### For bugs
- Rule Id (if any, e.g. SC1000):
  - https://github.com/koalaman/shellcheck/wiki/SC1066
  - https://github.com/koalaman/shellcheck/wiki/SC1068
- My shellcheck version (`shellcheck --version` or "online"): online
- [X] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [X] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related
  - https://github.com/koalaman/shellcheck/issues/874 is vaguely related, but covers `let`

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash
$variable = value   # this is a sh problem too
$array = (one two)
```

#### Here's what shellcheck currently says:

(Nothing.)

#### Here's what I wanted or expected to see:

- Warn about the dollar sign on the lhs of the assignment.
- Warn about the whitespace around the equals sign.

**Comment by tripleee** [comment]
Sorry, something in my browser was wrong.  Can't repro now.
